README.md

```diff
@@ -1,2 +1,132 @@
 # beaker-template
-Let's try and construct a Beaker DSL extension library!
+
+Let's construct a Beaker DSL extension library!
+
+# Why Would We Do Such a Thing?
+
+There are two reasons to create a Beaker library:
+
+1. To pull functionality out of Beaker core to be maintained/improved separately (mostly QE tasks)
+2. To provide additional functionality to their Beaker testing (biggest Beaker user use case)
```

Comment:
It is not clear what this point means. "To provide additional functionality to beaker" is what it is saying?